### PR TITLE
[feature]add dataloader

### PR DESCRIPTION
**Pull request by YuliangLiu0306**

### CODE REVIEW

**Comment by YuliangLiu0306** [comment]
image

File: titans/dataloader/imagenet/torchvision_imagenet.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        transforms.ToTensor(),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+    transform_test = transforms.Compose([
+        transforms.Resize(resize),
+        transforms.CenterCrop(crop),
+        transforms.ToTensor(),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+    data_root = os.environ['DATA']
```
Comment (FrankLeeeee) [inline titans/dataloader/imagenet/torchvision_imagenet.py:23]: This should be passed as a function parameter.

File: titans/dataloader/cifar10/torchvision_cifar10.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        transforms.ToTensor(),
+        transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)),
+    ])
+    transform_test = transforms.Compose([
+        transforms.Resize(resize),
+        transforms.ToTensor(),
+        transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)),
+    ])
+
+    train_dataset = CIFAR10(root=os.environ['DATA'], train=True, download=True, transform=transform_train)
```
Comment (FrankLeeeee) [inline titans/dataloader/cifar10/torchvision_cifar10.py:21]: `root` should be passes as a function paramter.

File: titans/dataloader/imagenet/dali_imagenet.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+import nvidia.dali.types as types
+import nvidia.dali.tfrecord as tfrec
+import torch
+import numpy as np
+from titans.dataloader.utils import RandAugment
+
+__all__ = ['DaliDataloaderWithRandAug', 'DaliDataloader']
+
+
+class DaliDataloaderWithRandAug(DALIClassificationIterator):
```
Comment (FrankLeeeee) [inline titans/dataloader/imagenet/dali_imagenet.py:13]: I would recommend having a `build_dali_imagenet_dataloader` for consistency as I see similar functions are given for the torchvision datasets.

**Review by FrankLeeeee — APPROVED** [review]

**Pull request merged**